recipes/edta/meta.yaml

```
@@ -21,45 +21,48 @@ requirements:
+    - tensorflow ==2.11 
+    - tesorter    
 
 test:
   commands:
     - EDTA.pl -h > /dev/null
-    - EDTA.pl --check_dependencies > /dev/null
-    - EDTA_processI.pl -h 2>&1 | grep -i "Perform EDTA"
-    - EDTA_raw.pl -h 2>&1 | grep -i "Obtain raw"
-    - lib-test.pl -h 2>&1 | grep -i "To test"
-    - LTR_FINDER_parallel -check_dependencies > /dev/null
+    #- EDTA.pl --check_dependencies > /dev/null 
```

yamllint

[error] trailing spaces

(trailing-spaces)